mizar/daemon/interface_service.py

```diff
@@ -163,21 +163,22 @@ def _ProvisionVethInterface(self, interface, cni_params):
         self.iproute.link('set', index=veth_peer_index, state='up', mtu=9000)
 
         # Configure the Transit Agent
-        self._ConfigureTransitAgent(interface)
+        self._ConfigureTransitAgent(interface, update)
 
-    def _ConfigureTransitAgent(self, interface):
+    def _ConfigureTransitAgent(self, interface, update):
         """
         Load the Transit Agent XDP program, program all the bouncer substrate,
         update the agent metadata and endpoint.
         """
         self.rpc.load_transit_agent_xdp(interface)
 
-        for bouncer in interface.bouncers:
-            self.rpc.update_agent_substrate_ep(
-                interface.veth.peer, bouncer.ip_address, bouncer.mac)
+        if update:
```

Comment:
This works because the bouncers make the update when they come up.
I originally had because the bouncers were not up upon host endpoint creation.
I can remove the flag and have this function be ran as is by having the endpoint create workflow retry if there are no bouncers  provisioned.